earthpy/tests/test_clip.py

```diff
@@ -19,33 +19,47 @@ def make_locs_gdf():
     return gdf
 
 
-def make_poly_in_gdf():
+def make_single_rect_poly_gdf():
     """ Bounding box polygon. """
     poly_inters = Polygon([(0, 0), (0, 10), (10, 10), (10, 0), (0, 0)])
     gdf = gpd.GeoDataFrame(
         [1], geometry=[poly_inters], crs={"init": "epsg:4326"}
     )
+    gdf["attr2"] = ["road"]
+    return gdf
+
+
+def make_smaller_clip_rect_poly_gdf():
+    """ Bounding box polygon. """
+    poly_inters = Polygon([(-5, -5), (-5, 15), (15, 15), (15, -5), (-5, -5)])
+    gdf = gpd.GeoDataFrame(
+        [1], geometry=[poly_inters], crs={"init": "epsg:4326"}
+    )
+    gdf["attr2"] = ["study area"]
     return gdf
 
 
 def make_locs_buff():
     """ Buffer points to create multi poly. """
     buffered_locations = make_locs_gdf()
     buffered_locations["geometry"] = buffered_locations.buffer(4)
+    buffered_locations["type"] = ["plot", "plot", "plot", "plot"]
```

Comment:
```suggestion
    buffered_locations["type"] = "plot"
```

That'll do the same thing more compactly

Reply:
it will. i again didn't realize i could do it that way and it would propagate to the entire DF!! thanks